pamelalozano/src/main/webapp/index.html

```diff
@@ -197,13 +199,16 @@ <h3>MongoDB</h3>
 </div>
 <div class="myContainer" id="comments">
     <form action="/data" method="POST">
-
       <p>Comment:</p>
       <input type="text" name="subject" placeholder="Subject">*<br /><br />
       <textarea type="text" name="msg" placeholder="Comment"></textarea>*
       <br/><br/>
-
     <input type="submit" />
+
+ <div id="comments-section">
+ 
```

Comment:
nit, can put the closing tag on the same line

Reply:
Done ! Thanks